bedrock/firefox/redirects.py

```diff
@@ -326,50 +406,87 @@ def firefox_mobile_faq(request, *args, **kwargs):
+    redirect(r'^projects/minefield(/.*)?$', '/firefox/channel/'),
     redirect(r'^projects/minimo(/.*)?$', 'https://wiki.mozilla.org/Mobile'),
     redirect(r'^projects/namoroka(/.*)?$', '/firefox/channel/'),
     redirect(r'^projects/nspr(?:/.*)?$', 'https://developer.mozilla.org/docs/NSPR'),
-    redirect(r'^projects/netlib(/.*)?$', 'https://developer.mozilla.org/en/Necko'),
-    redirect(r'^projects/plugins(/.*)?$',
-             'https://developer.mozilla.org/En/Plugins'),
+    redirect(r'^projects/netlib(/.*)?$',
+             'https://developer.mozilla.org/docs/Mozilla/Projects/Necko'),
+    redirect(r'^projects/plugins(/.*)?$', 'https://developer.mozilla.org/Add-ons/Plugins'),
     redirect(r'^projects/rt-messaging(/.*)?$', 'http://chatzilla.hacksrus.com/'),
     redirect(r'^projects/shiretoko(/.*)?$', '/firefox/channel/'),
     redirect(r'^projects/string(/.*)?$',
              'https://developer.mozilla.org/en/XPCOM_string_guide'),
     redirect(r'^projects/tech-evangelism(/.*)?$',
              'https://wiki.mozilla.org/Evangelism'),
     redirect(r'^projects/venkman(/.*)?$',
-             'https://developer.mozilla.org/en/Venkman'),
+             'https://developer.mozilla.org/docs/Archive/Mozilla/Venkman'),
     redirect(r'^projects/webservices/examples/babelfish-wsdl(/.*)?$',
-             'http://developer.mozilla.org/En/XML_Web_Services'),
-    redirect(r'^projects/xbl(/.*)?$', 'https://developer.mozilla.org/en/XBL'),
-    redirect(r'^projects/xforms(/.*)?$', 'https://developer.mozilla.org/en/XForms'),
-    redirect(r'^projects/xpcom(/.*)?$', 'https://developer.mozilla.org/en/XPCOM'),
+             'https://developer.mozilla.org/docs/SOAP_in_Gecko-based_Browsers'),
+    redirect(r'^projects/xbl(/.*)?$', 'https://developer.mozilla.org/docs/Mozilla/Tech/XBL'),
+    redirect(r'^projects/xforms(/.*)?$', 'https://developer.mozilla.org/docs/Archive/Web/XForms'),
+    redirect(r'^projects/xpcom(/.*)?$', 'https://developer.mozilla.org/docs/Mozilla/Tech/XPCOM'),
     redirect(r'^projects/xpinstall(/.*)?$',
-             'https://developer.mozilla.org/en/XPInstall'),
-    redirect(r'^projects/xslt(/.*)?$', 'https://developer.mozilla.org/en/XSLT'),
-    redirect(r'^projects/xul(/.*)?$', 'https://developer.mozilla.org/En/XUL'),
+             'https://developer.mozilla.org/docs/Archive/Mozilla/XPInstall'),
+    redirect(r'^projects/xslt(/.*)?$', 'https://developer.mozilla.org/docs/Web/XSLT'),
+    redirect(r'^projects/xul(/.*)?$', 'https://developer.mozilla.org/docs/Mozilla/Tech/XUL'),
     redirect(r'^quality/help(/.*)?$', 'http://quality.mozilla.org/get-involved'),
     redirect(r'^quality(/.*)?$', 'http://quality.mozilla.org/'),
 
     # Bug 654614 /blocklist -> addons.m.o/blocked
     redirect(r'^blocklist(/.*)?$', 'https://addons.mozilla.org/blocked/'),
 
+    redirect('^products/firebird$', '/firefox/products/'),
+    redirect('^products/firebird/download/$', '/firefox/new/'),
```

Comment:
Use `firefox.new` for consistency? Here and in following lines. Also for other internal redirects